src/main/java/Duke.java

```diff
@@ -34,35 +34,30 @@
     private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
     private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
 
-//    public static void main(String[] args) throws Exception {
-//        new Duke("data/duke.txt");
-//    }
+    public static void main(String[] args) throws Exception {
+
+    }
 
     /**
      * Constructs a Duke object which will initialise 1)storage, 2)tasks 3)Ui class..
      *
-     * @param filePath is the file directory for where duke.txt is saved
      */
-    public Duke(String filePath) throws Exception, DukeException {
-        storage = new Storage(filePath);
+    public Duke() throws Exception{
+        storage = new Storage("data/duke.txt");
         try {
             tasks = new TaskList(storage.readFile());
             ui = new Ui(tasks, storage);
         } catch (Exception e) {
```

Comment:
DukeException should be extending Exception; also if it's IOException that you're trying to catch, I think you should catch IOException e instead of Exception e?